docs/v1/openapi.yml

```diff
@@ -812,6 +812,102 @@ paths:
           $ref: "#/components/responses/UnauthorizedError"
         "404":
           $ref: "#/components/responses/DefaultError"
+    put:
+      tags:
+        - "QR code"
+      summary: used for update qrcode by id.
+      security:
+        - bearerAuth: [ ]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                show_id:
+                  type: string
+                  example: "f4f78cac-5db6-4ecc-ad13-5877705f3126"
+                episode_id:
+                  type: string
+                  example: "f4f78cac-5db6-4ecc-ad13-5877705f3126"
+                starts_at:
+                  type: string
+                  example: "2006-01-02T15:04:05Z"
+                  description: "RFC3339"
+                expires_at:
+                  type: string
+                  example: "2006-01-02T15:04:05Z"
+                  description: "RFC3339"
+                reward_amount:
+                  type: number
+                  example: 123.56
+      responses:
+        "200":
+          $ref: "#/components/responses/ResultSuccess"
+        "400":
+          $ref: "#/components/responses/DefaultError"
+        "401":
+          $ref: "#/components/responses/UnauthorizedError"
+        "404":
+          $ref: "#/components/responses/DefaultError"
+    delete:
+      tags:
+        - "QR code"
+      summary: used for delete qrcode by id.
+      security:
+        - bearerAuth: [ ]
+      responses:
```

Comment:
parameters:
        - name: id
          in: path
          description: QRcode ID.
          required: true
          schema:
            type: string